frontend/src/screens/researcher/studies/create/action-footer.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, React, useCallback, useState } from '@common';
+import { Link } from 'react-router-dom';
+import { Icon, ResearcherButton, Step } from '@components';
+
+export const ActionFooter: FC<{ step: Step, }> = ({ step }) => {
+    const [busy, setBusy] = useState(false)
+    const triggerAnimation = useCallback(() => {
+        setBusy(true)
+        const timer = setTimeout(() => {
+            setBusy(false)
+        }, 750);
+        return () => clearTimeout(timer);
+    }, [setBusy]);
```

Comment:
yeah the problem is the request was completing so fast that iris wanted to put in an artificial loading state so the users would actually see it and that it would be consistent haha

Reply:
not sure what the actual best approach here would be though